pkg/cmd/admin/registry/registry.go

```diff
@@ -74,7 +75,14 @@ type RegistryConfig struct {
 
 var errExit = fmt.Errorf("exit")
 
-const defaultLabel = "docker-registry=default"
+const (
+	defaultLabel = "docker-registry=default"
+	defaultPort  = 5000
+	// TODO: `/healthz` has been deprecated by `/`; keep it temporarily for
+	// backwards compatibility
+	healthzRoute               = "/healthz"
```

Comment:
> No, we support / endpoint since v3.1.1.0. I'll update the comment with the version.

Then the question about the transition remains.  Are you going to hardcode the tag where we branched?  Is there another way to introspect what's available?  Do we need to write something for a more complicated check?